specifications/sep-payment-channel-transactions.md

```diff
@@ -293,6 +299,14 @@ long as EI's sequence number n satisfies s_e <= n < s_i.  Because C_i has source
 account EI and sequence number s_i+1, D_i leaves EI in a state where C_i can
 execute.
 
+  D_i has a single signer in its `extraSigners` precondition that ensures that
+  if D_i is authorized, signed, and submitted, that the signature for C_i is
+  revealed. The signer is specified as:
+
+  - A ed25519 signed payload signer configured with:
+    - Payload set to the transaction hash of C_i.
+    - Public key set to R's signer.
```

Comment:
or `payee` might be more correct based off the above description
```suggestion
    - Public key set to other participant's signer.
```

Reply:
```suggestion
    - Public key set to payee's signer.
```